### ISSUE TITLE
Add keyboard navigation in left panel

### ISSUE THREAD

**Issue by megadix** [a85/POSTMan-Chrome-Extension#2300]
1. Postman Version: 4.6.2
2. App (Chrome app or Mac app): Mac app
3. OS details: Mac OS X 10.11.5 El Capitan
4. Is the Interceptor on and enabled in the app: No
5. Did you encounter this recently, or has this bug always been there: Always been there
6. Expected behaviour: when focus is on left panel (folders and scenarios) keyboard navigation should be enabled
7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):
8. Screenshots (if applicable)

Steps to reproduce the problem:
- add some collections, folders and calls
- select an element from the left panel
- hitting direction keys on the keyboard does not move between folders and calls

Suggested behavior:
- up/down keys: move up/down
- left: collapse folder
- right: expand folder

**Comment by a85** [comment]
@megadix +1. This is coming soon to Postman.

**Comment by a85** [comment]
This is available now in our native apps.
